kinto/plugins/accounts/views.py

```diff
@@ -127,3 +132,19 @@ def process_record(self, new, old=None):
             raise_invalid(self.request, **error_details)
 
         return new
+
+
+# Clear cache on account change
+@subscriber(ResourceChanged,
+            for_resources=('account',),
+            for_actions=(ACTIONS.UPDATE, ACTIONS.DELETE))
+def on_account_changed(event):
+    request = event.request
+    cache = request.registry.cache
+    settings = request.registry.settings
+    # Extract username and password from current user
+    username, password = extract_http_basic_credentials(request)
```

Comment:
Shouldn't it be `request.matchdict['id']` instead ? Since we make sure they match in the resource code?